internal/backends/python/python.go

```diff
@@ -394,10 +394,17 @@ func guess(python string) (map[api.PkgName]bool, bool) {
 			continue
 		}
 
-		pkg, ok := moduleToPypiPackage[modname]
-		if ok {
+		if modname[0] == '"' && modname[len(modname) - 1] == '"' {
```

Comment:
not sure i'm understanding this, what's the difference between a module with/without the surrounding quotes. Is this just to indicate the name is from a pragma?

Reply:
Here I come, barreling into code I've never read or written before, to ask if there's a more structured to indicate a package name comes from a pragma? Encoding it with quotes in the package name seems like it could lead to mistakes later on.